packages/ai/core/generate-text/stream-text.ts

```diff
@@ -1525,7 +1524,7 @@ However, the LLM results are expected to be small enough to not cause issues.
     }
 
     const state = createStreamingUIMessageState({
-      lastMessage,
+      lastMessage: structuredClone(lastMessage),
```

Comment:
Instead of cloning in `createStreamingUIMessageState` (which was breaking the state proxy for Svelte) we clone here